icu4c/source/test/intltest/measfmttest.cpp

```diff
@@ -4472,12 +4472,10 @@ void MeasureFormatTest::TestIdentifiers() {
         {"exbibyte", "exbibyte"},
         {"zebibyte", "zebibyte"},
         {"yobibyte", "yobibyte"},
-
         // Testing sort order of prefixes.
-        //
-        // TODO(icu-units#70): revisit when fixing normalization. For now we're
-        // just checking some consistency between C&J.
-        {"megafoot-mebifoot-kibifoot-kilofoot", "kibifoot-mebifoot-kilofoot-megafoot"},
+        {"megafoot-mebifoot-kibifoot-kilofoot", "mebifoot-megafoot-kibifoot-kilofoot"},
+        // Testing sort of units.
+        {"inch-foot-yard", "foot-inch-yard"}, // TODO: the result should be "yard-foot-inch"
```

Comment:
Are you sure? I don't think there's anything in the specification regarding the order of yard, foot, and inch.

(Spec: https://www.unicode.org/reports/tr35/tr35-info.html#Unit_Identifier_Normalization)

If I were you, I would treat it as out of scope for this pull request, and file a CLDR ticket for clarification of the specification.

Reply:
okay, good idea.